### ISSUE TITLE
Vector tile display format

### ISSUE THREAD

**Issue by QuangVidaGIS** [GeoNode/geonode#4407]
Im trying find the way to display my data as vector tile format. I try to import vector tile file as geojson to geonode, try to use WMS, WMTS, TMS service to connect and import vector tile data to geonode but all the layer just can display by PNG or JPEG format.
So how can i change the format display for Geonode

**Comment by t-book** [comment]
Have you seen that you can define the supported format in geoserver. https://docs.geoserver.org/latest/en/user/extensions/vectortiles/tutorial.html

**Comment by QuangVidaGIS** [comment]
Yes i have. But i want to display my layer in geonode. this guide seem like to build a WebGIS with html file

**Comment by t-book** [comment]
@QuangVidaGIS sorry for late response. the gihub issues are mainly for bugs. I would suggest using the following resource to ask for help:

- https://gitter.im/ (geonode channel)
- geonode user list: https://lists.osgeo.org/cgi-bin/mailman/listinfo/geonode-users
- geonode developer list: https://lists.osgeo.org/cgi-bin/mailman/listinfo/geonode-devel
